installers/vmware/main.go

```diff
@@ -46,11 +53,17 @@ func (i installer) BootScript(slug string) job.BootScript {
 	}
 
 	return func(ctx context.Context, j job.Job, s *ipxe.Script) {
-		script(j, s, path)
+		script(i, j, s, path)
 	}
 }
 
-func script(j job.Job, s *ipxe.Script, basePath string) {
+func script(i installer, j job.Job, s *ipxe.Script, basePath string) {
+	if i.extraIPXEVars != nil {
```

Comment:
[same comment](https://github.com/tinkerbell/boots/pull/281/files/344c786c99e2ece0c33702be22e98f6f48e9ff65#r896185299) about this loop

Reply:
Good catch, thank you!